HSPI/PageBuilder.cs

```diff
@@ -174,20 +174,24 @@ protected string HtmlTextBox(string name, string defaultText, int size, bool all
         /// </summary>
         /// <param name="name">The name used to create the references for the text box.</param>
         /// <param name="Checked">if set to <c>true</c> [checked].</param>
+        /// <param name="AutoPostBack">if set to <c>true</c></param>
+        /// <param name="SubmitForm">if set to <c>true</c></param>
         /// <returns>The text to insert in the web page to create the check box.</returns>
-        protected string BuildCheckBox(string name, bool Checked = false)
+        protected string BuildCheckBox(string Name, bool Checked = false, bool AutoPostBack = true, bool SubmitForm = true)
         {
-            return "<div id='" + name + "_div'>" + FormCheckBox(name, Checked) + "</div>";
+            return "<div id='" + Name + "_div'>" + FormCheckBox(Name, Checked, AutoPostBack, SubmitForm) + "</div>";
```

Comment:
All parameter names should be lower case to match the conventions in the rest of the codebade. Please change them back here and elsewhere in your PR.

I'll add some code conventions to the documentation soon.